app/Http/Controllers/AcceptancesController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Acceptance;
+use App\Submission;
+use Illuminate\Http\Request;
+
+class AcceptancesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $submission = Submission::findOrFail($request->input('submissionId'));
+
+        $acceptance = Acceptance::createFromSubmission($submission);
+
+        $submission->recordAcceptance($acceptance);
```

Comment:
I wonder if this would be reasonable to do within `Acceptance::createFromSubmission`?